pkg/operator/ceph/object/admin.go

```diff
@@ -29,15 +29,16 @@ type Context struct {
 	Context     *clusterd.Context
 	Name        string
 	ClusterName string
+	RunAsUser   string
```

Comment:
As a separate PR... We really need this RunAsUser property passed everywhere in the operator, rather than adding it just to the rgw context. How about a new ClusterContext type that has the `namespace`, `clusterd.Context`, and `RunAsUser`. The first two are already passed around everywhere so would be good to consolidate these.
```
struct ClusterContext {
  clusterd.Context // declare this inline
  Namespace string
  RunAsUser string
}
```
But we likely should wait until closer to 1.4 since this will create terrible merge conflicts.

Reply:
Yes, let's wait further down 1.4.